#include "Kiax2DotTelDialog.h"Kiax2DotTelDialog::Kiax2DotTelDialog(QWidget* parent) : QDialog(parent){ 	ui.setupUi(this);	ui.dialButton->setEnabled(false);	connect(ui.dialButton, SIGNAL(pressed()), this, SLOT(dialButtonClicked()));	connect(ui.cancelButton, SIGNAL(pressed()), this, SLOT(cancelButtonClicked()));	connect(ui.numbersList, SIGNAL(itemActivated ( QListWidgetItem *)), this, SLOT(dialButtonClicked()));	connect(ui.numbersList, SIGNAL(currentItemChanged(QListWidgetItem*, QListWidgetItem*)), 							this, SLOT(currentItemChanged(QListWidgetItem*, QListWidgetItem*)));#if defined(MACXDEP)	setWindowFlags(Qt::Sheet); #endif		ui.infoTextLabel->setText("");} Kiax2DotTelDialog::~Kiax2DotTelDialog(){}void Kiax2DotTelDialog::dialButtonClicked(){	accept();}void Kiax2DotTelDialog::cancelButtonClicked(){	reject();}void Kiax2DotTelDialog::addTelNumber(QString number, QString description){	ui.numbersList->addItem(description);
	numbers.push_back(number);	ui.numbersList->setCurrentRow(0); // select first number}void Kiax2DotTelDialog::currentItemChanged(QListWidgetItem* current, QListWidgetItem* previous){	if(ui.numbersList->currentRow()>=0)	{		ui.dialButton->setEnabled(true);	}}QString Kiax2DotTelDialog::getNumber(){	if (ui.numbersList->currentItem()!=NULL)		return numbers[ui.numbersList->currentRow()];	else return QString("");
}void Kiax2DotTelDialog::addTelDescription(QString descr){	//description += descr + "\n";	ui.infoTextLabel->setText(ui.infoTextLabel->text() + descr + QString("\n"));}
